pkg/cmd/cli/cmd/edit.go

```diff
@@ -59,14 +78,24 @@ saved copy to include the latest resource version.`
 
 // NewCmdEdit implements the OpenShift cli edit command
 func NewCmdEdit(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
-	filenames := []string{}
+	var options EditOptions
+	options.fullName = fullName
 	cmd := &cobra.Command{
 		Use:     "edit (RESOURCE/NAME | -f FILENAME)",
 		Short:   "Edit a resource on the server",
 		Long:    editLong,
 		Example: fmt.Sprintf(editExample, fullName),
 		Run: func(cmd *cobra.Command, args []string) {
-			err := RunEdit(fullName, f, out, cmd, args, filenames)
+			options.cmd = cmd
+			options.args = args
+
+			if err := options.Complete(); err != nil {
+				cmdutil.CheckErr(err)
+			}
+			if err := options.Validate(); err != nil {
+				cmdutil.CheckErr(err)
+			}
+			err := options.RunEdit()
```

Comment:
Do we have a documented preference for one of

```go
if err := ...; err != nil {
  ...
}
```

or

```go
err := ...
if err != nil {
  ...
}
```

over the other? Granted, here we have `err == errExit` rather than `err != nil`, but the style difference stands out nevertheless between these consecutive checks.